hash/m20.xml

```diff
@@ -456,8 +461,8 @@ license:CC0-1.0
 		</part>
 	</software>
 
-
 <!-- This is a user disk with high school math programs -->
+
 	<software name="m20utent" supported="no"> <!-- Collection of BASIC programs, boot PCOS and run basic to use -->
```

Comment:
Please don’t separate the comment from the item it belongs to like this.

Reply:
done